### PR TITLE
Translated working-with-starters file in Marathi

### PR DESCRIPTION
**Pull request by Abdulhussainjarif**

### CODE REVIEW

File: docs/docs/recipes/working-with-starters.md [hunk 1]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
+[Starters](/docs/starters/) बॉयलरप्लेट गॅटस्बी साइट आहेत त्यांची अधिकृत देखभाल केली जाते, किंवा समुदायाद्वारे.
 
-## Using a starter
+## स्टार्टर वापरणे
 
-### Prerequisites
+### पूर्व शर्ती
 
-- The [Gatsby CLI](/docs/gatsby-cli) installed
+- द [Gatsby CLI](/docs/gatsby-cli) स्थापित करा
```
Comment (spielers) [inline docs/docs/recipes/working-with-starters.md]: You can use word installed directly No need to convert it

File: docs/docs/recipes/working-with-starters.md [hunk 2]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
 
-### Directions
+### दिशानिर्देश
```
Comment (spielers) [inline docs/docs/recipes/working-with-starters.md]: दिशानिर्देश-> सूचना

File: docs/docs/recipes/working-with-starters.md [hunk 3]
```diff
@@ -3,36 +3,36 @@ title: "Recipes: Working with Starters"
...
 
-1. Find the starter you'd like to use. (_The [Starter Library](/starters/?v=2) is a good place to look!_)
+1. आपण वापरू इच्छित असलेले स्टार्टर शोधा. (द [Starter Library](/starters/?v=2) पाहण्याची चांगली जागा आहे!_)
```
Comment (spielers) [inline docs/docs/recipes/working-with-starters.md]: remove  द and आहे

**Review by spielers — CHANGES_REQUESTED** [review]
Do fix the suggested!

Best,
spielers

File: docs/docs/recipes/working-with-starters.md (hunk 3 above)
Comment (Abdulhussainjarif) [reply docs/docs/recipes/working-with-starters.md]: @spielers I have made all the suggested changes, please merge my PR. 
 Also, Let me know if you need any further changes!

Thanks & Regards,
Abdulhussain Jarif

**Review by spielers — APPROVED** [review]
Hey, @Abdulhussainjarif  

Thanks

Best,
spielers

**Pull request merged**